crates/iota-core/src/authority.rs

```diff
@@ -1593,7 +1593,7 @@ impl AuthorityState {
         let transaction_data = &certificate.data().intent_message().value;
         let (kind, signer, gas) = transaction_data.execution_parts();
 
-        #[expect(unused_mut)]
+        #[cfg_attr(not(any(msim, fail_points)), expect(unused_mut))]
```

Comment:
What is fail_points?

Reply:
I guess it's a config attribute used for this compilation variant. It's also suppressed in the main `Cargo.toml`.

Reply:
But where is it defined? I can't find it

Reply:
It's used for the macro `fail_point_if`

Reply:
I see it being used there but it had to be enabled somewhere? And that's what I can't find

Reply:
It's enabled by the person who compiles.

Reply:
Alright, just find it weird that no script uses it and no documentation mentions it :(